README.md

```diff
@@ -152,7 +146,7 @@ To see the latest list of the default dependencies that are injected, check out
 List of external dependencies used and exposed by spur-web. They can be found at npmjs.org using their original names.
 
 | Name               | Original Module Name                                             |
-| :----              | :----                                                            |
+| :----------------- | :--------------------------------------------------------------- |
```

Comment:
Did prettier add these? `:----` is a markdown shortcut to make it dynamic and not have to keep track of the length as you write. Can you please revert?